using_images/other_images/jenkins.adoc

```diff
@@ -119,64 +119,67 @@ Once selected, you must link:../../dev_guide/templates.html#creating-resources-f
 
 == Using Jenkins as a Source-To-Image builder
 
-If you want to customize the official OpenShift Jenkins image, you can use
-Docker layering or you can use the image as a Source-To-Image builder.
+To customize the official OpenShift Jenkins image, you have two options:
+
+* Use Docker layering.
+* Use the image as a Source-To-Image builder, described here.
+
 You can use link:../../architecture/core_concepts/builds_and_image_streams.html#source-build[S2I]
 to copy your custom Jenkins Jobs definitions, additional
-plugins or replace the provided `config.xml` file with your own, custom, configuration.
+plugins or replace the provided *_config.xml_* file with your own, custom, configuration.
 
 In order to include your modifications in the Jenkins image, you need to have a Git
-repository, with the following directory structure:
+repository with the following directory structure:
+
+*_plugins_*::
+This directory contains binary Jenkins plugins you want to copy into Jenkins.
```

Comment:
@bfallonf Good catch, thanks.  After some pondering i think the most accurate word in that position would be "those":

"This directory contains those binary Jenkins plugins you want to copy into Jenkins."